tests/phpunit/elementor/core/experiments/test-manager.php

```diff
@@ -39,11 +37,17 @@ public function test_add_feature() {
 			'name' => 'test_feature',
 			'state' => Experiments_Manager::STATE_DEFAULT,
 			'on_state_change' => null,
+			'new_site' => [
+				'default_active' => false,
+				'always_active' => false,
+				'minimum_installation_version' => null,
+			],
+			'mutable' => true,
```

Comment:
It's not in the `$allowed_options` list, how can it work?

Reply:
It's added to the feature by the `add_feature` method. Not by the user

Reply:
So, Why do you set `'mutable' => true,`? it should not affect the feature.

Reply:
This is the expected results. Not the data that is sent to the feature